arbitrum-docs/partials/_local-devnet-flags.md

```diff
@@ -0,0 +1,22 @@
+<section class='small-table'>
+| Flag                                     | Description                                                                                                                                                                                                                                                                                     |
+| ---------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `--build` | rebuild docker images                                                                                                                                                          |
+| `--dev`                             | build nitro and blockscout dockers from source instead of pulling them. Disables simple mode                                                                                                                                                                                                 |
```

Comment:
```suggestion
| `--dev`                             | Builds Nitro and Blockscout Dockers from source instead of pulling them. Disables simple mode                                                                                                                                                                                                 |
```